From aa76cde2dc599e18bc4ac17a84056ac156c06ee3 Mon Sep 17 00:00:00 2001
From: Orel Eliyahu <orel_e@eyal-emi.co.il>
Date: Wed, 15 Jan 2025 14:24:49 +0200
Subject: [PATCH] fix size LPDDR TO 1GB

---
 include/configs/imx8mp_evk.h | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/include/configs/imx8mp_evk.h b/include/configs/imx8mp_evk.h
index a26c651dcf5..a5e91426082 100644
--- a/include/configs/imx8mp_evk.h
+++ b/include/configs/imx8mp_evk.h
@@ -199,13 +199,13 @@
 /* Totally 6GB DDR */
 #define CONFIG_SYS_SDRAM_BASE		0x40000000
 #define PHYS_SDRAM			0x40000000
-#define PHYS_SDRAM_SIZE			0xC0000000	/* 3 GB */
-#define PHYS_SDRAM_2			0x100000000
-#ifdef CONFIG_TARGET_IMX8MP_DDR4_EVK
-#define PHYS_SDRAM_2_SIZE		0x40000000	/* 1 GB */
-#else
-#define PHYS_SDRAM_2_SIZE		0xC0000000	/* 3 GB */
-#endif
+#define PHYS_SDRAM_SIZE			0x40000000	/* 3 GB */
+//#define PHYS_SDRAM_2			0x100000000
+//#ifdef CONFIG_TARGET_IMX8MP_DDR4_EVK
+//#define PHYS_SDRAM_2_SIZE		0x40000000	/* 1 GB */
+//#else
+//#define PHYS_SDRAM_2_SIZE		0xC0000000	/* 3 GB */
+//#endif
 
 #define CONFIG_MXC_UART_BASE		UART2_BASE_ADDR
 
